lib/doconce/html.py

```diff
@@ -1243,8 +1243,8 @@ def html_code(filestr, code_blocks, code_block_types,
         """
         if 'label' in tex_blocks[i]:
             # Fix label -> \label in tex_blocks
-            tex_blocks[i] = tex_blocks[i].replace(' label{', ' \\label{')
-            tex_blocks[i] = re.sub(r'^label\{', '\\label{', tex_blocks[i],
+            tex_blocks[i] = tex_blocks[i].replace(r' label{', r' \\label{')
```

Comment:
This is a normal string replacement (not using `re`), so we should not escape the backslash when using a raw string.
```
tex_blocks[i] = tex_blocks[i].replace(r' label{', r' \label{')
```

Reply:
Only the two first arguments need to have well-defined escape characters. The string which we perform the replacements on (in this case `tex_blocks[i]`) can have unknown escapes.

Reply:
agreed